charts/karpenter/templates/webhooks.yaml

```diff
@@ -19,17 +19,6 @@ webhooks:
     failurePolicy: Fail
     sideEffects: None
     rules:
-      - apiGroups:
```

Comment:
In order for this to work properly with the Knative reconciliation mechanism, we would need to update the addition of the defaulting webhook for `AWSNodeTemplate`s on this line: https://github.com/aws/karpenter/blob/main/pkg/webhooks/webhooks.go#L44. We _can_ do this but we are planning to remove the entire batch of webhooks in the near-future.

Is this currently blocking anything or causing issues? I'm wondering if we can wait to remove this until we remove all of the webhooks outright.

Reply:
Ah ok no worries not blocking. We could wait.

Reply:
If it's alright with you, I'm going to close this since we won't take this right now. You can track the removal of the webhooks through: https://github.com/aws/karpenter-core/issues/103